ecosystem/sep-0011.md

```diff
@@ -172,11 +174,17 @@ A few aggregate values are special-cased:
 
 The `Asset` type is rendered as a string in one of two forms:
   - `native` (or any string up to 12 characters not containing an unescaped colon) for the native asset.
-  - `Code:IssuerAccountID` for issued assets.
+  - `Code:IssuerAccountID` for issued assets, i.e. assets of type
+    `ASSET_TYPE_CREDIT_ALPHANUM4` and `ASSET_TYPE_CREDIT_ALPHANUM12`.
 
     "Code" must consist of printable ASCII characters (octets 0x21 through 0x7e).
 
     The sequence `\x` introduces a hex escape sequence, e.g., `\x00` to introduce a 0-valued byte.  Otherwise, `\` escapes the next character, so `\\` is required to introduce a backslash. Stellar disallows assets of type `ASSET_TYPE_CREDIT_ALPHANUM12` that have fewer than 5 bytes, but such assets can be represented in binary XDR, and so in txrep are rendered with trailing `\x00` (escaped NUL bytes) to as to make the length 5---e.g., the 12-byte asset code `ABC` is rendered `ABC\x00\x00`. Note that stellar-core disallows non ASCII bytes in `AssetCode` fields, so the primary use of this feature is to construct or examine invalid transaction test cases.
+  - `PoolID:POOL` for pool shares, i.e.  `ASSET_TYPE_POOL_SHARE`.
```

Comment:
```suggestion
  - `PoolID:POOL` for liquidity pool shares, i.e. assets of type `ASSET_TYPE_POOL_SHARE`.
```

Reply:
That does beg the question, maybe the suffix should be `:POOLSHARE`. @jonjove Are there likely to be other forms of POOL assets?

Reply:
> the XDR type is actually `PoolID` and the asset type is formally `POOL_SHARE`

Refs:
- https://github.com/stellar/stellar-core/blob/dc5f5a392098b82bd9453a2aa4259e7af600ad9d/src/xdr/Stellar-ledger-entries.x#L17
- https://github.com/stellar/stellar-core/blob/dc5f5a392098b82bd9453a2aa4259e7af600ad9d/src/xdr/Stellar-ledger-entries.x#L240-L241

Reply:
I suggested "liquidity" because we had the exact same discussion for the Horizon API, and we concluded that being explicit here was better. Although the XDR naming is more general, it refers to liquidity pools.

Reply:
>XDR naming is more general

I agree, the right thing to do in Horizon or applications is probably to be more explicit. Or possible we should have been more explicit in the XDR in the first place.

SEP-11 is a string representation of the XDR though, so I think it should mirror the XDR. SEP-11 should bring as little as possible to the XDR.

Reply:
>But it's worth mentioning that typedefs don't correspond to different types just different names, so if we had LiquidityPoolID and OtherPoolID both map to the same type (Hash) then they would be compatible anyway.

The property you describe is true of XDR as its defined, and I'm guessing our C++ implementation, but it's not practically true in all programming languages, so that property doesn't flow downstream.

Regardless, naming, and communication, are more important to me than type identity, or convert-ability. For future definitions I think there is more value in the consistency of names, e.g. `ASSET_TYPE_LIQUIDITY_POOL_SHARE`.

Given that they are always liquidity pools, I've changed the format to a `:LP` suffix, which as it happens was @stanford-scs's original suggestion.